components/apps_form/apps_form_component.css

```diff
@@ -0,0 +1,3 @@
+.modal-body-opacity {
```

Comment:
This is a pretty generic name for a class. The class name should be specific to the Apps Form component, since this css is actually shared globally across the whole application. Maybe something like this opacity class name is already in this application?

Reply:
changed name to `apps-form-modal-body-opacity`